project-base/assets/js/commands/translations/findLang.test.js

```diff
@@ -0,0 +1,15 @@
+import findLang from './findLang';
+
+test.each([
+    ['', undefined],
+    ['lang', undefined],
+    ['lang.po', 'lang'],
+    ['lang.en.po', 'en'],
+    ['lang.js', 'lang'],
```

Comment:
method name tells me it finding language (presumably from filename). Now I'm confused, is it ok to return `lang` from `lang.js`? Can you please explain the reason to me?

Reply:
5a34a254c46c71fdf33b29d212d0a09b896ddae0